src/Persistence/Yaml/SerializerBuilderExtensions.cs

```diff
@@ -24,6 +24,7 @@ public static SerializerBuilder WithFirstClassModels(this SerializerBuilder buil
     public static DeserializerBuilder WithFirstClassModels(this DeserializerBuilder builder)
     {
         return builder
+            .IgnoreUnmatchedProperties()
```

Comment:
I can approve for now but we need to discuss this

Reply:
the issue here is if customer makes mistake in Yaml we will quietly ignore it instead of reporting it as an error